posts/2021-05-11-database-ssl-primer.adoc

```diff
@@ -183,93 +191,98 @@ Property Documentation: https://jdbc.postgresql.org/documentation/head/connect.h
     <variable name="oracle.keystore" value="javax.net.ssl.keystore=security/keystore.p12;javax.net.ssl.trustStoreType=PCKS12;javax.net.ssl.keyStorePassword=${KEYSTORE_PASSWORD};"/>
     <variable name="oracle.truststore" value="javax.net.ssl.trustStore=security/truststore.p12;javax.net.ssl.trustStoreType=PCKS12;javax.net.ssl.trustStorePassword=${TRUSTSTORE_PASS};"/>
     <variable name="oracle.conn.props.store" value="${oracle.tls.props}${oracle.truststore}${oracle.keystore}" />
-    
+
     <dataSource jndiName="jdbc/oracle-tls-truststore" jdbcDriverRef="jdbcLib">
     	<properties.oracle URL="${env.SSL_URL}" connectionProperties="${oracle.conn.props.store}" />
     </dataSource>
 
 </server>
 ----
 
-**Enable TLS**
+- Enable TLS
+
+To enable TLS connections on the Oracle JDBC driver, add the `connectionProperties="oracle.net.ssl_version=1.2;"` attribute to the `dataSource properties` element.
+
```

Comment:
This part doesn't line up with the example- In the example, the `oracle.net.ssl_version=1.2` value is not set in `connectionProperties`, but in `variable name="oracle.tls.props" value="`.

Reply:
Yes, I believe that most oracle customers would use properties to configure the `connectionProperties` attribute as it is a long list of semicolon deliminated properties.  It should be easy to understand that those variables are just substituted in at runtime.

Reply:
ah, ok, I see now that variable is reused in the connection properties